public/index.php

```diff
@@ -25,12 +25,16 @@
 ini_set("display_errors", 1);
 session_start();
 header('Content-Type: text/html; charset=utf-8');
+//Sanitize $_POST parameters
+$post_input  = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
 
 $templates = new League\Plates\Engine('../templates');
 
+$post_input['oauth_consumer_key'] = filter_input(INPUT_POST,'oauth_consumer_key',FILTER_SANITIZE_NUMBER_INT);
```

Comment:
I can see about contacting Instructure to get a specific character set. If not we can just use the generic FILTER_SANITIZE_STRING to get at least the baseline sanitization.